sunpy/map/tests/test_mapsequence.py

```diff
@@ -196,3 +197,30 @@ def test_norm_animator(hmi_test_map):
 def test_map_sequence_plot(aia171_test_map, hmi_test_map):
     seq = sunpy.map.Map([aia171_test_map, hmi_test_map], sequence=True)
     seq.plot()
+
+
+def test_save(aia171_test_map, hmi_test_map, tmp_path):
+    """
+    Tests the MapSequence save function
+    """
+    seq = sunpy.map.Map([aia171_test_map, hmi_test_map], sequence=True)
+
+    with pytest.raises(ValueError, match="{index} must be specified"):
+        seq.save(f"{tmp_path.as_posix()}_index:03", filetype='fits', overwrite=True)
+    seq.save(f"{tmp_path.as_posix()}_{{index:03}}.fits", filetype='auto', overwrite=True)
+
+    test_seq = sunpy.map.Map(f"{tmp_path.as_posix()}_000.fits",
+                             f"{tmp_path.as_posix()}_001.fits", sequence=True)
```

Comment:
```suggestion
    with pytest.raises(ValueError, match="'{index}' must be appear in the string"):
        seq.save("index", filetype='fits', overwrite=True)

    base_str = (tmp_path / "map").as_posix()
    seq.save(f"{base_str}_{{index:03}}.fits", filetype='auto', overwrite=True)

    test_seq = sunpy.map.Map(f"{base_str}_000.fits",
                             f"{base_str}_001.fits", sequence=True)
```
You should be creating files within the `tmp_path` directory.

Reply:
Oh yeah, my bad.